sites/default/statement.inc.php

```diff
@@ -104,6 +104,12 @@ function create_HTML_statement($stmt) {
   #minimum_amount_due_to _print
   if ($stmt['amount'] <= ($GLOBALS['minimum_amount_to_print']) && $GLOBALS['use_statement_print_exclusion']) return "";
   if ($stmt['statement_print'] == "NO" && $GLOBALS['use_statement_print_exclusion']) return "";
+  if ($GLOBALS['use_statement_print_exclusion'] && $GLOBALS['insurance_statement_exclude'] !=4 ){
+   if ($GLOBALS['insurance_statement_exclude'] ==0 && $stmt['insurance_no_statement_print_pri']) return "";
```

Comment:
Could use better {} syntax, and nested statements.  Not sure if the 4 enum values cover the behavior needed.

Reply:
May need more for combinations.

Reply:
Terry, does this satisfy the real-world need, or do we need those other combos?
Can the user override these controls at the actual printing UI?

Reply:
The actual UI for printing is pretty dumb . The request was to not print statements for a specific insurance company IE medicare. There is also another request being worked on to make the UI a little more intelligent and offer more printing abilities.